V1/Interfaces/Services/IEncryptionService.cs

```diff
@@ -0,0 +1,27 @@
+using System.Threading.Tasks;
+// TODO: All references to WalletContract below will change to reference the "new" version
+using CoviIDApiCore.V1.DTOs.Wallet;
+// TODO: All references to CovidTestCredentialParameters below will change to reference the "new" version
+using CoviIDApiCore.V1.DTOs.Credentials;
+
+namespace CoviIDApiCore.V1.Interfaces.Services
+{
+    public interface IEncryptionService
```

Comment:
Shouldn't it be crypto service. Encryption implies only encryption happens and not decryption

Reply:
Sure, I'll change this.